[2023-01-25 23:14:50,567] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Respuesta_Confirmacion_NASA scheduled__2023-01-01T00:00:00+00:00 [queued]>
[2023-01-25 23:14:50,581] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Respuesta_Confirmacion_NASA scheduled__2023-01-01T00:00:00+00:00 [queued]>
[2023-01-25 23:14:50,581] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-01-25 23:14:50,582] {taskinstance.py:1377} INFO - Starting attempt 3 of 5
[2023-01-25 23:14:50,582] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-01-25 23:14:50,603] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): Respuesta_Confirmacion_NASA> on 2023-01-01 00:00:00+00:00
[2023-01-25 23:14:50,609] {standard_task_runner.py:52} INFO - Started process 3438 to run task
[2023-01-25 23:14:50,616] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Respuesta_Nasa', 'Respuesta_Confirmacion_NASA', 'scheduled__2023-01-01T00:00:00+00:00', '--job-id', '462', '--raw', '--subdir', 'DAGS_FOLDER/00-ETL_spaceX.py', '--cfg-path', '/tmp/tmp3k3qc_z7', '--error-file', '/tmp/tmpt6q6kcbp']
[2023-01-25 23:14:50,617] {standard_task_runner.py:80} INFO - Job 462: Subtask Respuesta_Confirmacion_NASA
[2023-01-25 23:14:50,699] {task_command.py:371} INFO - Running <TaskInstance: Respuesta_Nasa.Respuesta_Confirmacion_NASA scheduled__2023-01-01T00:00:00+00:00 [running]> on host fd9891b969b0
[2023-01-25 23:14:50,797] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=Respuesta_Nasa
AIRFLOW_CTX_TASK_ID=Respuesta_Confirmacion_NASA
AIRFLOW_CTX_EXECUTION_DATE=2023-01-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-01T00:00:00+00:00
[2023-01-25 23:14:50,798] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-01-25 23:14:50,799] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 20 && echo "ConfirmaciÃ³n de la NASA, pueden proceder" > /tmp/response_20230101.txt']
[2023-01-25 23:14:50,811] {subprocess.py:85} INFO - Output:
[2023-01-25 23:15:10,815] {subprocess.py:96} INFO - Command exited with return code 0
[2023-01-25 23:15:10,856] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Respuesta_Nasa, task_id=Respuesta_Confirmacion_NASA, execution_date=20230101T000000, start_date=20230125T231450, end_date=20230125T231510
[2023-01-25 23:15:10,872] {local_task_job.py:221} WARNING - State of this instance has been externally set to success. Terminating instance.
[2023-01-25 23:15:10,875] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 3438. PIDs of all processes in the group: [3438]
[2023-01-25 23:15:10,875] {process_utils.py:80} INFO - Sending the signal Signals.SIGTERM to group 3438
[2023-01-25 23:15:10,883] {process_utils.py:75} INFO - Process psutil.Process(pid=3438, status='terminated', exitcode=0, started='23:14:50') (3438) terminated with exit code 0
